packages/network/wireguard/modules-load.d/wireguard.conf

```diff
@@ -0,0 +1 @@
+wireguard
```

Comment:
It might be better to load the kernel module with `ExecStartPre=-` in service to not load it when service is not used.